app/plugins/Files/js/loaders.js

```diff
@@ -76,7 +89,9 @@ function uploadDir(dirPath, nickname) {
 					notify('Cannot read ' + file, 'error');
 					return;
 				}
-				if (!isUnixHiddenPath(filePath) && stats.isFile()) {
+				if (file.slice(-4) === '.sia') {
+					loadDotSia(filePath);
```

Comment:
Iunno... I think apps generally do expected conveniences like this. In what case would someone want to load a .sia file?